src/main/java/com/uwetrottmann/tmdb2/entities/Account.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2017 Nikolas Mavropoylos
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.uwetrottmann.tmdb2.entities;
+
+public class Account {
+
+    public static class Avatar {
+        public static class GRAvatar {
```

Comment:
For instance in C# every interface starts with a Capitalized I, every public accessible field/property also starts with a Capitalized Letter, protected and private fields are lower cased, and internal fields are prefixed with "_". 

In C# we do a lot of Linq, a way better version of Java's Streams, which introduced in Java 8, The code is way different any many cases, as Java lacks of many C# features, which you have to thing a lot to implement them in java.

My highest disappointment of java was the lacking of operator overloading, and generic definition restrictions. Many things could go differently on this project having these kind of features. Later this year i might make a Port of tmdb2-java, to C#. If i do i will let you know to see all the kind of differences, and the beauty of the usage of these features.